odinApp/src/odinDetector.cpp

```diff
@@ -324,17 +329,97 @@ void OdinDetector::report(FILE *fp, int details) {
 asynStatus OdinDetector::drvUserCreate(asynUser *pasynUser,
                                        const char *drvInfo,
                                        const char **pptypeName,
-                                       size_t *psize) {
-  return ADDriver::drvUserCreate(pasynUser, drvInfo, pptypeName, psize);
+                                       size_t *psize)
+{
+  static const char *functionName = "drvUserCreate";
+  asynStatus status = asynSuccess;
+  int index;
+  RestParam *generatedParam = 0;
+  std::string value;
+
+  // Accepted parameter formats for HTTP parameters
+  //
+  // ODI_...  => Integer parameter
+  // ODE_...  => Enum parameter
+  // ODS_...  => String parameter
+  // ODD_...  => Double parameter
+  if (findParam(drvInfo, &index) && strlen(drvInfo) > 4 && strncmp(drvInfo, "OD", 2) == 0 &&
+      drvInfo[3] == '_') {
+
+    if (mUserParameters.count(drvInfo) == 0){
```

Comment:
Is this necessary? It will already not allow using the same asyn parameter name twice.